.github/workflows/hermetic_library_generation.yaml

```diff
@@ -0,0 +1,29 @@
+name: Hermetic library generation upon generation config change through pull requests
+
+on:
+  pull_request:
+    types:
+    - synchronize
+    paths:
+    - generation_config.yaml
```

Comment:
I think this means this Github action is only triggered if there is a PR changes the `generation_config.yaml`?

Reply:
That is my intention. I'll verify its actual behavior.